### DISCUSSION TITLE
Enhanced RTMP V2 (Alpha) Gets Advanced Audio!

### DISCUSSION THREAD

**Discussion by veovera** [veovera/enhanced-rtmp#28] [Announcements]
## Upgrades to Enhanced RTMP

In response to valuable feedback from our users, we are excited to announce a significant upgrade to the latest alpha specification, "[enhanced-rtmp-v2](https://veovera.github.io/enhanced-rtmp/docs/enhanced/enhanced-rtmp-v2)" (or "[enhanced-rtmp-v2.pdf](https://veovera.github.io/enhanced-rtmp/docs/enhanced/enhanced-rtmp-v2.pdf)"). This update emphasizes audio enhancements, advancing our commitment to the ongoing development of the Real-Time Messaging Protocol (RTMP). The Enhanced RTMP initiative now includes:

- **Advanced Audio Codecs:** Integrates codecs such as AC-3, E-AC-3, Opus, and FLAC, meeting diverse audio quality and compression needs and ensuring compatibility with modern systems.
- **Multichannel Audio Configurations:** Supports multichannel audio to enhance auditory experiences without compromising existing setups.
- **Advanced Video Codecs:** Introduces codecs like VP8, VP9, HEVC, and AV1, with HDR support to meet modern display and content standards.
- **Video Metadata:** Expands VideoPacketType.Metadata to support a broader range of video metadata types.
- **FourCC Signaling:** Includes FourCC signaling for advanced codecs as well as legacy codecs such as AVC, AAC, and MP3.
- **Multitrack Capabilities:** Allows concurrent management and processing of multiple audio and video streams, enhancing media experiences.
- **Reconnect Request Feature:** Improves connection stability and resilience with a new Reconnect Request feature.

Moved by the insights and suggestions from our community, we accelerated the audio upgrade to ensure RTMP remains aligned with contemporary streaming technology standards, maintaining its relevance and effectiveness in today's digital landscape.

We invite developers who implement this specification to inform us about their reference implementations or solutions. Your feedback is crucial in verifying the effectiveness of the specification and identifying any issues that need to be addressed. This collaborative effort helps us ensure the robustness and reliability of the Enhanced RTMP standard.

Together, we can shape the future of media streaming technology, ensuring that RTMP remains innovative. Your participation and feedback are crucial to our ongoing endeavors.

For further details or to participate in the alpha testing, please visit https://veovera.org/.

## Alpha Version Disclaimer

To draw attention to alpha disclaimer in [enhanced-rtmp-v2](https://veovera.github.io/enhanced-rtmp/docs/enhanced/enhanced-rtmp-v2):

This document outlines the alpha version of the Enhanced Real-Time Messaging Protocol (E-RTMP) v2. As we continue to refine and enhance the protocol, we remain open to implementing necessary updates based on user feedback and further testing. While there is a possibility of introducing breaking changes during the alpha stage, we are committed to maintaining the integrity of the General Availability (GA) versions and strive to ensure they remain free from breaking changes.

Engaging with the alpha version provides a unique opportunity to influence the final specifications of E-RTMP v2. We value your input and look forward to collaborating with you on this exciting journey towards developing a more robust and efficient protocol.

**Comment by bayareaunicorn** [comment]
Thank you for everyone's work and contributions at keeping RTMP a relative and innovative technology.
